dozer-sql/expression/src/scalar/string.rs

```diff
@@ -589,4 +784,74 @@ mod tests {
             );
         }
     }
+
+    fn test_chr(val: i64) {
+        let row = Record::new(vec![]);
+
+        // Field::String
+        let mut value = Box::new(Literal(Field::Int(val)));
+        assert_eq!(
+            evaluate_chr(&Schema::default(), &mut value, &row).unwrap(),
+            Field::String((((val % 256) as u8) as char).to_string())
```

Comment:
The test is the same as the implementation, which means it doesn't really test anything